edit-post/components/visual-editor/style.scss

```diff
@@ -67,7 +67,7 @@
 
 // This is a focus style shown for blocks that need an indicator even when in an isEditing state
 // like for example an image block that receives arrowkey focus.
-.edit-post-visual-editor .editor-block-list__block:not( .is-selected )   {
+.edit-post-visual-editor .editor-block-list__block:not( .is-selected ):not( [data-type="core/paragraph"] )   {
```

Comment:
Why not paragraphs?

Reply:
Good question. This branch has been around long enough that I don't recall adding this — could it have come in as part of a rebase and later been removed? 

In any case I removed it again becaues I couldn't see any regressions as a result.